lib/barbeque/message_handler/job_execution.rb

```diff
@@ -20,9 +20,15 @@ def run
           raise DuplicatedExecution.new(e.message)
         end
 
-        stdout, stderr, status = run_command
+        begin
+          stdout, stderr, status = run_command
+        rescue Exception => e
```

Comment:
Any kind of exception could be raised from the runner in theory and the job_execution status should not remain pending in any case.